src/RequestInsurance/HttpResponse.php

```diff
@@ -45,6 +49,18 @@ public function isTimedOut(): bool
         return isset($this->connectException);
```

Comment:
Har du undersøgt hvad det her betyder, og om den også skal have `$this->requestException` med i betragtningen?